ansible/workers.yml

```diff
@@ -106,14 +139,10 @@
       - openscap-devel
```

Comment:
To install openscap-devel  on RHEL 7 you need to have Optional repository enabled.

Reply:
I think so. See the difference between rhel-7-server-optional-rpms disabled and enabled below:
```
[root@localhost ~]# yum repolist
Loaded plugins: product-id, search-disabled-repos, subscription-manager
repo id                                                                                            repo name                                                                                                 status
rhel-7-server-rpms/7Server/x86_64                                                                  Red Hat Enterprise Linux 7 Server (RPMs)                                                                  23,347
repolist: 23,347
[root@localhost ~]# yum install openscap-devel
Loaded plugins: product-id, search-disabled-repos, subscription-manager
rhel-7-server-rpms                                                                                                                                                                          | 3.5 kB  00:00:00     
No package openscap-devel available.
Error: Nothing to do
[root@localhost ~]# yum install --enablerepo rhel-7-server-optional-rpms openscap-devel
Loaded plugins: product-id, search-disabled-repos, subscription-manager
rhel-7-server-optional-rpms                                                                                                                                                                 | 3.3 kB  00:00:00     
(1/3): rhel-7-server-optional-rpms/7Server/x86_64/group                                                                                                                                     |  31 kB  00:00:00     
(2/3): rhel-7-server-optional-rpms/7Server/x86_64/updateinfo                                                                                                                                | 2.3 MB  00:00:00     
(3/3): rhel-7-server-optional-rpms/7Server/x86_64/primary_db                                                                                                                                | 7.7 MB  00:00:00     
Resolving Dependencies
--> Running transaction check
---> Package openscap-devel.x86_64 0:1.2.17-2.el7 will be installed
--> Finished Dependency Resolution

Dependencies Resolved

===================================================================================================================================================================================================================
 Package                                           Arch                                      Version                                          Repository                                                      Size
===================================================================================================================================================================================================================
Installing:
 openscap-devel                                    x86_64                                    1.2.17-2.el7                                     rhel-7-server-optional-rpms                                    1.5 M

Transaction Summary
===================================================================================================================================================================================================================
Install  1 Package

Total download size: 1.5 M
Installed size: 18 M
Is this ok [y/d/N]: N
Exiting on user command
Your transaction was saved, rerun it with:
 yum load-transaction /tmp/yum_save_tx.2019-01-08.02-49.FIgVkW.yumtx
[root@localhost ~]# 
```

Reply:
I noticed you already mention 
> In case of RHEL, enable the `rhel-7-server-extras-rpms` and `rhel-7-server-optional-rpms` repos using e.g. the `subscription-manager`.

 so I think that's OK.